go.mod

```diff
@@ -215,3 +215,5 @@ go 1.22.6
 retract v0.0.2
 
 replace github.com/btcsuite/btclog/v2 => github.com/ellemouton/btclog/v2 v2.0.0-20241210110018-997ee6596623
+
+replace github.com/lightningnetwork/lnd/fn/v2 => github.com/starius/lnd/fn/v2 v2.0.7-optcg
```

Comment:
for future: it might be easier just to point to the commit hash in your repo. That way it is also easy to check that it matches the commit hash in the other PR. Then you dont need to push your own tag :)